lib/wallet/walletdb.js

```diff
@@ -503,15 +503,15 @@ class WalletDB extends EventEmitter {
     if (this.feeRate > 0)
       return this.feeRate;
 
-    const rate = await this.client.estimateFee(blocks);
+    const fee = await this.client.estimateFee(blocks);
```

Comment:
What happens if fee is `null`?

Reply:
dang, thanks. I modified it so 
```
if (!rate || rate < this.network.feeRate)
  return this.network.feeRate;
```
and tested by intentionally breaking return values from `NodeClient.estimateFee()`, walletDB falls back to the network `feeRate`

Reply:
You sure?

Reply:
This will throw an error if the result of `this.client.estimateFee()` is `null`, and a `null` response can not be ruled out.
```
const {rate} = await this.client.estimateFee(blocks);
```

Reply:
```
    const json = await this.client.estimateFee(blocks);

    if (!json || json.rate < this.network.feeRate)
      return this.network.feeRate;

    if (json.rate > this.network.maxFeeRate)
      return this.network.maxFeeRate;

    return json.rate;
```
?